Helm 3 Install Failure

Using helm 3:  Version:"v3.2.1"
With kuttl v0.2.0
with a kind cluster (originally reported on GKE cluster)

## kuttl setup
setup:
folder:  e2e/helm
file 00-setup.yaml

cat e2e/helm/00-setup.yaml 
```
apiVersion: kudo.dev/v1alpha1
kind: TestCase
commands:
- command: helm install jenkins stable/jenkins
timeout: 300 
```

## helm setup
`helm repo add stable https://kubernetes-charts.storage.googleapis.com`

test run locally:  `helm install jks stable/jenkins`
works fine... 

## failure

cat kuttl-test.yaml 
```
apiVersion: kudo.dev/v1beta1
kind: TestSuite
parallel: 4
```

run from master
```
go run cmd/kubectl-kuttl/main.go test e2e/
=== RUN   kuttl
    kuttl: harness.go:339: starting setup
    kuttl: harness.go:225: running tests using configured kubeconfig.
    kuttl: harness.go:297: running tests
    kuttl: harness.go:67: going to run test suite with timeout of 30 seconds for each step
=== RUN   kuttl/harness
=== RUN   kuttl/harness/helm
=== PAUSE kuttl/harness/helm
=== CONT  kuttl/harness/helm
    kuttl/harness/helm: logger.go:41: 19:44:14 | helm | Creating namespace: kudo-test-still-hermit
    kuttl/harness/helm: logger.go:41: 19:44:14 | helm/0-setup | starting test step 0-setup
    kuttl/harness/helm: case.go:156: the server could not find the requested resource
    kuttl/harness/helm: case.go:159: failed in step 0-setup
    kuttl/harness/helm: logger.go:41: 19:44:15 | helm | helm events from ns kudo-test-still-hermit:
    kuttl/harness/helm: case.go:149: the server could not find the requested resource
    kuttl/harness/helm: logger.go:41: 19:44:15 | helm | Deleting namespace: kudo-test-still-hermit
    kuttl: harness.go:326: run tests finished
--- FAIL: kuttl (0.80s)
    --- FAIL: kuttl/harness (0.00s)
        --- FAIL: kuttl/harness/helm (0.28s)
FAIL
exit status 1
```

helm fails with `the server could not find the requested resource` under these conditions

This has *nothing* to do with helm:
```
$ go run cmd/kubectl-kuttl/main.go test e2e/
=== RUN   kuttl
=== RUN   kuttl/harness
=== RUN   kuttl/harness/helm
=== PAUSE kuttl/harness/helm
=== CONT  kuttl/harness/helm
--- FAIL: kuttl (6.21s)
    harness.go:339: starting setup
    harness.go:219: running tests with a mocked control plane (kube-apiserver and etcd).
    harness.go:198: started test environment (kube-apiserver and etcd) in 6.170285237s, with following options:
        --advertise-address=127.0.0.1
        --etcd-servers={{ if .EtcdURL }}{{ .EtcdURL.String }}{{ end }}
        --cert-dir={{ .CertDir }}
        --insecure-port={{ if .URL }}{{ .URL.Port }}{{ end }}
        --insecure-bind-address={{ if .URL }}{{ .URL.Hostname }}{{ end }}
        --secure-port={{ if .SecurePort }}{{ .SecurePort }}{{ end }}
        --disable-admission-plugins=ServiceAccount,NamespaceLifecycle
        --service-cluster-ip-range=10.0.0.0/24
        --advertise-address={{ if .URL }}{{ .URL.Hostname }}{{ end }}
    harness.go:297: running tests
    harness.go:67: going to run test suite with timeout of 30 seconds for each step
    --- FAIL: kuttl/harness (0.00s)
        --- FAIL: kuttl/harness/helm (0.01s)
            logger.go:41: 08:58:40 | helm | Creating namespace: kudo-test-cheerful-macaw
            logger.go:41: 08:58:40 | helm/0-setup | starting test step 0-setup
            case.go:156: the server could not find the requested resource
            case.go:159: failed in step 0-setup
            logger.go:41: 08:58:40 | helm | helm events from ns kudo-test-cheerful-macaw:
            case.go:149: the server could not find the requested resource
            logger.go:41: 08:58:40 | helm | Deleting namespace: kudo-test-cheerful-macaw
    harness.go:326: run tests finished
    harness.go:434: tearing down mock control plane
FAIL
exit status 1
$ cat e2e/helm/00-setup.yaml 
apiVersion: kudo.dev/v1alpha1
kind: TestCase
commands:
- command: ls
timeout: 300
$ 
```

--- a/pkg/test/utils/kubernetes.go
+++ b/pkg/test/utils/kubernetes.go
@@ -321,7 +321,7 @@ func Namespaced(dClient discovery.DiscoveryInterface, obj runtime.Object, namesp
 	resource, err := GetAPIResource(dClient, obj.GetObjectKind().GroupVersionKind())
 	if err != nil {
 
-		return "", "", err
+		return "", "", fmt.Errorf("retrieving API resource for %v failed: %v", obj.GetObjectKind().GroupVersionKind(), err)
 	}
 
 	if !resource.Namespaced {

